src/views/v2/rsvp/ari/form/fields/meta.php

```diff
@@ -26,6 +28,10 @@
  */
 $has_meta = apply_filters( 'tribe_tickets_rsvp_has_meta', $rsvp, false );
 
+if ( $rsvp instanceof Tribe__Tickets__Ticket_Object ) {
```

Comment:
Is there any scenario where `$rsvp` wouldn't be `Tribe__Tickets__Ticket_Object`? It'd be great if we can move this to the RSVP object rather than having the logic in the templates. Or just doing the early return in the entry point instead of here. What do you think?

Reply:
You're right this was meant for `$has_meta`

```suggestion
if ( $has_meta instanceof Tribe__Tickets__Ticket_Object ) {
```